regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -14,11 +14,18 @@ var CommentEvents = require('../../events/comment-events');
 var AttachmentView = require('../../views/comment/attachment-view');
 var comments = require('../../collections/comment-collection');
 
+/**
```

Comment:
I'm going to be really obnoxious here and point out that afaik, jsdoc-style comments should have a leading space after the first line:

```javascript
/**
 * foo
 */
```

not

```javascript
/**
* bar
*/
```